src/content/release-notes.mdx

```diff
@@ -5,6 +5,32 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.17.0
+
+23 January 2024
+
+This versions is compatible with Kubernetes versions 1.25 to 1.28.
+
+### New Features
+
+- Enable search in namespace resources UI  <!-- 7577 -->
+- Enable admins to [transfer namespace ownership](core/components/namespaces/#transfer-namespace-ownership)  <!-- 7593 -->
+- Add [Pod metrics to Okteto Insights](admin/okteto-insights/#pod-metrics)  <!-- 7594 -->
+
+### Improvements
+
+- Hoist Personal Access Tokens menu from Setup to Settings  <!-- 7574 -->
+- Rename Launch → Deploy for all UI  <!-- 7576 -->
+- Rename Secrets -> Variables under both Settings and Admin sections  <!-- 7581 -->
+- Support multi-line secrets  <!-- 7580 -->
+- Improve buildkit cache reloading on helm upgrades  <!-- 7615 -->
```

Comment:
@pchico83 Do you think it is worth to let users know that this upgrade will imply a restart in the buildkit  cache?

Reply:
Yes, I would mention it